src/main/java/net/wurstclient/altmanager/MicrosoftLoginManager.java

```diff
@@ -0,0 +1,335 @@
+package net.wurstclient.altmanager;
+
+import com.google.gson.*;
+import net.minecraft.client.util.Session;
+import net.wurstclient.WurstClient;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLDecoder;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class MicrosoftLoginManager {
+    private static String clientID = "00000000402b5328";

```

Comment:
Is this some kind of special/default `clientID`? I have seen the same on used in [MiniLauncher](https://github.com/MiniDigger/MiniLauncher/blob/master/launcher/src/main/java/me/minidigger/minecraftlauncher/launcher/gui/MsaFragmentController.java#L33), but every other `clientID` generated by Azure looks totally different. How does this work?

Reply:
It is the Minecraft client id